### PR TITLE
Rename boar

### PR DESCRIPTION
**Pull request by zachgibb**

### CODE REVIEW

**Comment by ericgibb** [comment]
closes #260

**Comment by ericgibb** [comment]
also updates the formatting of HOARD token:

image

File: src/screens/Wallet/Wallets/WalletListEntry.js [hunk 1]
```diff
@@ -28,6 +28,9 @@ const WalletListEntry = ({
       true
     ) && (Number(price) * Number(balance)).toFixed(2);
 
+  const formattedPrice = Number(price)
+    .toFixed(5)
+    .replace(/0{0,5}$/, '');
```
Comment (zachgibb) [inline src/screens/Wallet/Wallets/WalletListEntry.js]: we should truncate only the final 3 zeros, to leave us with something like `$X.00`

Comment (zachgibb) [reply src/screens/Wallet/Wallets/WalletListEntry.js]: 👍

**Comment by ericgibb** [comment]
closed in #291

**Pull request merged**